components/vf-banner/vf-banner--inline.njk

```diff
@@ -1,3 +1,5 @@
+<h2>This component has been <span style="color: rgb(228, 0, 70);">deprecated</span>. Please use the <a class="vf-link" href="./vf-banner--info/"new</a> component.</h2>
```

Comment:
Better to wrap this in `<!-- -->` as this could silently blow up a production site?

Reply:
Ah!

Reply:
Noticed the href was broken, just fixed that.